packages/backend-data/src/js_resolver_handler.ts

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable */
```

Comment:
Top-level passthrough resolver request/response handler (see: https://docs.aws.amazon.com/appsync/latest/devguide/resolver-reference-overview-js.html#anatomy-of-a-pipeline-resolver-js)

It's required for defining a pipeline resolver. The only purpose it serves is returning the output of the last function in the pipeline back to the client.

Reply:
If we change the `any` to `unknown` (or a minimal object type for the response) does eslint stop complaining?

Reply:
It mainly complains about missing JSDocs for both functions. This one actually gets deployed into customers' APIs, but we can still add something boilerplate there. Will update